house/migrations/0003_delete_job_alter_city_id_alter_country_id_and_more.py

```diff
@@ -0,0 +1,49 @@
+# Generated by Django 4.0.3 on 2022-03-27 11:43
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('house', '0002_add_initial_data'),
+    ]
+
+    operations = [
+        migrations.DeleteModel(
+            name='Job',
```

Comment:
What does it mean? 
what is Job?

Reply:
It refers to the "Job" class from house.models